scripts/prepare-release.sh

```diff
@@ -5,7 +5,7 @@ this=`basename $0`
 
 usage () {
 cat << EOF
-Usage: $this [-h] RELEASE_VERSION
+Usage: $this [-h] RELEASE_VERSION GPG_KEY GPG_KEYRING
```

Comment:
So OWNERS (who wish to do a release) are required to generate a GPG key first ?
should this GPG key be associated with the github account ?

Reply:
Yes, it should. I don't know if there's a way to verify this 🤔

Reply:
Thanks for trying this out @adrianchiris!

Yes, I used the key uid reported by `gpg --list-secret-keys`

I.e. something like
```
./scripts/prepare-release.sh v0.8.0 'Markus Lehtonen <markus.lehtonen@intel.com>' ~/.gnupg/secring.gpg
```

Reply:
@adrianchiris: added an example to `prepare-release.sh` usage string

Reply:
Mmm... did exactly the same but it complains the key was not found even though i see it in my secret keyring.

maybe i have some env issue.

Reply:
Hmm, strange 🤔 Don't put the `[ultimate]` there, if you have it

Reply:
Ach, actually the `secring.gpg` file is obsolete and used by older versions of gpg. You prolly don't have your key there

Reply:
Update the script even more notes